src/Bot.py

```diff
@@ -0,0 +1,48 @@
+import discord
+import json
+from discord.ext import commands
+
+f = open("../config.json")
+config = json.loads(f.read())
```

Comment:
Since discord token is a sensitive variable, it's highly preferable to read them from environment variables.

Reply:
changed the config storage to a .env file